src/main/java/com/google/netpcapanalysis/interfaces/models/PCAP.java

```diff
@@ -0,0 +1,9 @@
+package com.google.netpcapanalysis.interfaces.models;
+
+import java.net.InetAddress;
+import java.util.*;
+
+public interface PCAP {
```

Comment:
PCAP has more to it than IP addresses. You should choose a different name for the interface. Melba has a PCAPData class that contains all the n tuples in a record, so a PCAP file is a list of PCAPData objects. So basically you should try to use that if you can, by creating a list of PCAP objects. You might only use the IP field that is fine and your mock would only mock IP.

Reply:
Can I use PCAP as a stub, and let Melba merge on top so we can use hers?

Reply:
@mnuzen Are you okay with merging your class into the interface?